test/e2e/test_helpers.go

```diff
@@ -146,12 +147,14 @@ type testHelperInterface interface {
 	getNutanixResourceIdentifierFromEnv(envVarKey string) infrav1.NutanixResourceIdentifier
 	getNutanixResourceIdentifierFromE2eConfig(variableKey string) infrav1.NutanixResourceIdentifier
 	getVariableFromE2eConfig(variableKey string) string
+	getDefaultStorageContainerNameAndUuid(ctx context.Context) (string, string, error)
```

Comment:
nit
```suggestion
	getDefaultStorageContainerNameAndUUID(ctx context.Context) (string, string, error)
```

Reply:
@dlipovetsky Is it really important for test helper? There are examples for both naming conventions in the interface.

Reply:
Not important; that's why I called it a "nit" :smile: